project/Dependencies.scala

```diff
@@ -210,13 +210,13 @@ object Dependencies {
     libraryDependencies ++= Seq(
         "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
         "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttpVersion,
-        "com.pauldijou" %% "jwt-core" % JwtCoreVersion, //ApacheV2
+        "com.pauldijou" %% "jwt-core" % "3.1.0", //ApacheV2
```

Comment:
Consider bumping the version for the entire Alpakka project? The latest is 4.2.0. The jump from 3 to 4 is apparently ["not really a breaking change release."](https://github.com/pauldijou/jwt-scala/releases/tag/v4.0.0)